Spigot-Server-Patches/0083-Allow-Reloading-of-Custom-Permissions.patch

```diff
@@ -19,9 +19,18 @@ index 5b91630..ab18e1a 100644
 +    public void reloadPermissions() {
 +        ((SimplePluginManager) pluginManager).clearPermissions();
 +        loadCustomPermissions();
++        for (Plugin plugin : pluginManager.getPlugins()) {
++            plugin.getDescription().getPermissions().forEach((perm) -> {
```

Comment:
If leaving the exception unhandled isn't much of an issue, this can be simplified to:
`plugin.getDescription().getPermissions().forEach(pluginManager::addPermission);`